app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/ui/screens/codePager/CodePagerScreen.kt

```diff
@@ -45,12 +45,11 @@ import com.esri.arcgismaps.kotlin.sampleviewer.R
 import com.esri.arcgismaps.kotlin.sampleviewer.model.DefaultSampleInfoRepository
 import com.esri.arcgismaps.kotlin.sampleviewer.model.Sample
 import com.esri.arcgismaps.kotlin.sampleviewer.ui.components.CodeView
-import com.esri.arcgismaps.kotlin.sampleviewer.ui.components.MarkdownTextView
+import com.esri.arcgismaps.kotlin.sampleviewer.ui.components.ReadmeView
 
 /**
- * This class shows both ReadMe and Code Previews for each sample
+ * Shows both README and Code for each sample.
  */
-
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun CodePagerScreen(
```

Comment:
Would this be a better name to use? Swift calls this screen a [SampleInfoView](https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/main/Shared/Supporting%20Files/Views/SampleInfoView.swift)
```suggestion
fun SampleInfoScreen(
```